openlibrary/templates/site/body.html

```diff
@@ -22,8 +22,10 @@
       $for flash in get_flash_messages():
         <div class="$flash.type"><span>$flash.message</span></div>
       </div>
-      <div class="page-banner page-banner-body">
-        $:announcement
-      </div>
+      $if show_announcement:
+        $ announcement = '<strong>New Feature:</strong> You can now embed Open Library books on your website! &nbsp; <a href="https://blog.openlibrary.org/2018/05/06/turn-your-website-into-a-library/" data-ol-link-track="AnnouncementsBar|BlogWidget" class="btn primary">Learn More</a>'
+        <div class="page-banner page-banner-body">     
+          $:announcement
```

Comment:
Also remove the `announcement` variable, as well. You can replace the `$:announcement` line with its html value.

Reply:
If we're hiding the announcement because it is out of date, then the conditional should be on the announcement and no boolean should be needed.
```
$announcement = ''
$if announcement:
   <div class="page-banner page-banner-body">
     $:announcement
```

Reply:
I think that @cdrini suggested to use the boolean to easily bring back the announcement, just in case.  A compromise could be to check the announcement string as you suggested and keep an example commented out.

```
$# Sample announcement
$# <strong>Title:</strong> Description &nbsp; <a href="#" data-ol-link-track="AnnouncementsBar|BlogWidget" class="btn primary">Learn More</a>
$announcement = ''
$if announcement:
  <div class="page-banner page-banner-body">     
    $:announcement
  </div>
```

Reply:
sorry to make you jump through hoops -- this solution sounds good to me, happy to merge once we get this and the newline squared away. Thank you for your help on this!

Reply:
That's OK, don't worry about it :) I'm not a big fan of commented out stuff, but in this case it could be a good compromise IMHO.